sungjin/app.js

```diff
@@ -0,0 +1,91 @@
+require("dotenv").config();
+
+const express = require("express");
+const cors = require("cors");
+const logger = require("morgan");
+const { DataSource } = require("typeorm");
+
+const appDataSource = new DataSource({
+  type: process.env.DB_CONNECTION,
+  host: process.env.DB_HOST,
+  port: process.env.DB_PORT,
+  username: process.env.DB_USERNAME,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
+});
+appDataSource.initialize().then(() => {
+  console.log("Data Source has been initialized!");
+});
+const app = express();
+
+app.use(cors());
+app.use(logger("dev"));
+app.use(express.json());
+app.use(express.json());
+app.get("/ping", function (req, res, next) {
+  res.json({ message: "pong" });
```

Comment:
- 엔드포인트는 app.use 와는 속성이 다른 코드의 새로운 전개라고 보여집니다.
- 따라서 한 줄의 라인 브레이킹으로 서로다른 코드의 영역 분리를 시켜주세요!

```suggestion
app.use(express.json());

app.get("/ping", function (req, res, next) {
  res.json({ message: "pong" });
  
```